config/helm/chart/default/templates/Common/logagent/serviceaccount-logagent.yaml

```diff
@@ -0,0 +1,21 @@
+{{- include "dynatrace-operator.platformRequired" . }}
+# Copyright 2021 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: dynatrace-logagent
```

Comment:
since the dynakube field is `logModule` should we use that name here as well? potentially for clusterrole/serviceaccount/... names as well